core/sql/common/Ipc.cpp

```diff
@@ -4580,44 +4565,21 @@ IpcServerClass::IpcServerClass(IpcEnvironment *env,
   }
   if (allocationMethod_ == IPC_ALLOC_DONT_CARE)
     {
-#if (defined(NA_GUARDIAN_IPC))
-	  // NA_WINNT is set and NA_GUARDIAN_IPC is set
-	  // The standard method on NT is to create a Guardian process
-	  // in order to run in an NT only or simulated environment we can set an environment
-	  // variable to override that mechanism.
-	  if (getenv("SQL_NO_NSK_LITE") == NULL)
-	  {
-	   allocationMethod_ = IPC_LAUNCH_GUARDIAN_PROCESS;
-	  }
-	  else 
-	  {
-	   allocationMethod_ = IPC_LAUNCH_NT_PROCESS;
-          time_t tp;
-          time(&tp);
-	  nextPort_ = IPC_SQLESP_PORTNUMBER + tp % 10000; // arbitrary
-	  };
-
-#else // NA_GUARDIAN_IPC
-
-	  // NA_WINNT is set and NA_GUARDIAN_IPC is NOT set
-	  // The standard method on NT is to create an NT process
-	  // We use socket based communication but launch the process ourself rather
-	  // than using INETD.
-	  // Eventually we will use nsk lite to create new processes and to
-	  // communicate via the message system.
-	  // in order to run in an NT only or simulated environment we can set an environment
-	  // variable to override that mechanism.
-	  if (getenv("SQL_NO_NSK_LITE") == NULL)
-	  {allocationMethod_ = IPC_LAUNCH_GUARDIAN_PROCESS;
-	  }
-	  else 
-	  {
-	  allocationMethod_ = IPC_LAUNCH_NT_PROCESS;
+      // NA_WINNT is set and NA_GUARDIAN_IPC is set
```

Comment:
Perhaps delete this one line of the comment since the defines are no longer used?